src/functionaltest/java/com/ericsson/ei/frontend/TestBridgeCurlFunctions.java

```diff
@@ -96,6 +85,16 @@ public void testHandleSubscriptionsUserSpecifiedBackend() throws Exception {
         mockClient1.verifyZeroInteractions();
     }
 
+    private void setupMockEndpoints() {
```

Comment:
Public before private, even if the public is tagged After.

Reply:
Done.